.gitignore

```diff
@@ -9,3 +9,4 @@ node_modules/
 *.vsix
 *.tsbuildinfo
 hello-world
+generated/
```

Comment:
We don't want to exclude changes to the `generated` files in this repo, as that would make (eventually) it impossible to bootstrap the Langium grammar. It also serves as a regression test for changes in the `langium-cli` package.